python/googleapis/python-multi/Dockerfile

```diff
@@ -75,6 +75,13 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/* \
   && rm -f /var/cache/apt/archives/*.deb
 
+# Install Microsoft ODBC 17 Driver and unixodbc for testing SQL Server samples
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+RUN curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list > /etc/apt/sources.list.d/mssql-release.list
+RUN apt-get update
+RUN ACCEPT_EULA=Y apt-get install -y msodbcsql17
+RUN apt-get install unixodbc-dev
```

Comment:
Can you chain these into a single command?

Also use the following as in the other examples to minimize image size:
```
  && apt-get clean autoclean \
  && apt-get autoremove -y \	  && apt-get autoremove -y \
  && rm -rf /var/lib/apt/lists/* \	  && rm -rf /var/lib/apt/lists/* \
  && rm -f /var/cache/apt/archives/*.deb	  && rm -f /var/cache/apt/archives/*.deb
```